Kitodo/src/main/webapp/WEB-INF/templates/includes/system/indexing.xhtml

```diff
@@ -0,0 +1,137 @@
+                                <p:graphicImage id="create-mapping-success" alt="success" value="/pages/images/success.svg" style="max-height: 20px;" rendered="#{!indexingForm.indexingInProgress() and indexingForm.indexState == 'MAPPING_SUCCESS'}"/>
+                            </td>
+                        </tr>
+                        <!-- Delete Index -->
+                        <tr>
+                            <td colspan="2">
+                                <p:commandButton widgetVar="deleteIndex"
+                                                 id="deleteIndexButton"
+                                                 rendered="#{SecurityAccessController.isAdminOrHasAuthorityGlobal('editIndex')}"
+                                                 value="#{msgs.deleteIndex}"
+                                                 style="width: 100%;"
+                                                 action="#{indexingForm.deleteIndex(true)}"
+                                                 disabled="#{!indexingForm.indexExists() or indexingForm.indexingInProgress()}"
+                                                 update="indexingTable"
+                                                 onclick="return confirm('Do you REALLY want to delete the whole index?');deleteIndex(#{indexingForm.objectTypesAsJson});"/>
+                            </td>
+                            <td>
+                                <p:graphicImage id="delete-index-loading" alt="in progress" value="/pages/images/ajax-loader.gif" style="max-height: 20px; display: none;"/>
+                                <p:graphicImage id="delete-index-error" alt="failed" value="/pages/images/fail.svg" style="max-height: 20px;" rendered="#{!indexingForm.indexingInProgress() and indexingForm.indexState == 'DELETE_ERROR'}"/>
+                                <p:graphicImage id="delete-index-success" alt="success" value="/pages/images/success.svg" style="max-height: 20px;" rendered="#{!indexingForm.indexingInProgress() and indexingForm.indexState == 'DELETE_SUCCESS'}"/>
+                            </td>
+                        </tr>
+                        </tbody>
+                    </table>
+                    <table id="indexingTable">
+                        <thead>
+                        <tr>
+                            <th><h:outputText value="#{msgs.objectType}"/></th>
+                            <th><h:outputText value="#{msgs.startIndexing}"/></th>
+                            <th><h:outputText value="#{msgs.progress}"/></th>
+                            <th><h:outputText value="#{msgs.indexedEntries}"/></th>
+                            <th><h:outputText value="#{msgs.status}"/></th>
+                        </tr>
+                        </thead>
+                        <tbody>
+
+                        <ui:repeat value="#{indexingForm.objectTypes}" var="objectType">
+                            <ui:include src="../../../../pages/inc_indexing/indexingRow.xhtml">
```

Comment:
Why did you use a relative path here? In all other places is a absolute path used.